src/app/component/general/eco-news/eco-news-detail/eco-news-detail.component.ts

```diff
@@ -6,10 +6,9 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./eco-news-detail.component.css']
 })
 export class EcoNewsDetailComponent implements OnInit {
+  idNumberFromEmit: string;
```

Comment:
Please add acess modificators such as private or public

Reply:
Fixed, has to be deleted this variable